web/src/components/Both.js

```diff
@@ -101,6 +104,7 @@ export class Both extends Component {
                         <option value="box">Box</option>
                     </select>
                     </label>
+                    {/* CR * sub components to group form fields */}
```

Comment:
Your form doesn't have to be one component. It can be made up of several combined components. This will also help reusability in other places. ♻️